src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.csproj

```diff
@@ -25,6 +25,9 @@
     <Reference Include="System.Windows.Forms" />
     <Reference Include="System.Xaml" />
     <Reference Include="WindowsBase" />
+    <Reference Include="Microsoft.Practices.Prism">
+        <HintPath>..\..\extern\prism\Microsoft.Practices.Prism.dll</HintPath>
```

Comment:
should this be copy local false?

Reply:
it was not set before. I think it could be
I just restored it as it was